src/Transport/Sending/DefaultBatchedOperationsToBrokeredMessagesConverter.cs

```diff
@@ -155,7 +158,10 @@ void ApplyDeliveryConstraints(BrokeredMessage brokeredMessage, BatchedOperation
 
         static void CopyHeaders(OutgoingMessage outgoingMessage, BrokeredMessage brokeredMessage)
         {
-            brokeredMessage.Properties[Headers.ReplyToAddress] = brokeredMessage.ReplyTo;
+            if (!string.IsNullOrEmpty(brokeredMessage.ReplyTo))
```

Comment:
So if brokeredMessage.ReplyTo is empty, we don't want even to have a header (empty)?